cli/service/service.go

```diff
@@ -4,6 +4,14 @@ import (
 	"bufio"
 	"encoding/json"
 	"fmt"
+	"io"
```

Comment:
Please use `make format` to format source code so that these kinds of changes don't happen and the imports will remain in alphabetic order.

Reply:
I tried make format but it did not restore the imports back to alphabetic order.   Thus I reverted it manually.   
btw, the original order change was done automatically by VSCode golang source format, I guess it was trying to organize the imports so that default packages would be on top.